app/models/user.rb

```diff
@@ -43,6 +43,22 @@ def minor
     age < 18
   end
 
+  def rounded_average_spending(months = 2)
```

Comment:
I would like to make some kind of button to go to the bank instant (without the need to enter how much you want to put in you account). If we do this with a fixed amount it could result in a much too low amount for some users and a much too high amount for other user. I agree that this might a bit too aggressive, I think we should discuss this on the next meeting